### PR TITLE
Feature/1087 use arrow keys to navigate through timepicker sliders

### PR DESCRIPTION
**Pull request by danielsteinkogler**
## Basics

- [ ] I added a line to [changelog.md](/doc/changelog.md)
- [ ] The PR is rebased with current master.
- [ ] Details of what you changed are in commit messages.
- [ ] References to issues, e.g. `close #X`, are in the commit messages and changelog.
- [ ] The buildserver is happy.

## Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
- [ ] I fixed all affected documentation
- [ ] I fixed the introduction tour
- [ ] I wrote migrations in a way that they are compatible with already present data
- [ ] I fixed all affected decisions
- [ ] I added automated tests or a [manual test protocol](../doc/tests/manual/protocol.md)
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)
- [ ] Exceptions to any guidelines are documented

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] I've read through the whole documentation
- [ ] I've checked conformity to guidelines

### CODE REVIEW

File: frontend/src/features/map_planning/components/timeline/TimelineDatePicker.tsx [hunk 1]
```diff
@@ -203,6 +207,51 @@ const TimelineDatePicker = ({ onSelectDate, onLoading, defaultDate }: TimelineDa
...
+        case 'day':
+          setFocusedSlider('year');
+          break;
+        default:
+          break;
+      }
+    }
+  };
+
   useEffect(() => {
```
Comment (Bushuo) [inline frontend/src/features/map_planning/components/timeline/TimelineDatePicker.tsx:255]: Can this be broken out into an event handler function?
The state change is somewhere invoked, there should also be the place where the callbacks are called.
Also please justify when using eslint-disable-next-line react-hooks/exhaustive-deps.

**Comment by markus2330** [comment]
jenkins build please

**Comment by markus2330** [comment]
jenkins build please

**Comment by danielsteinkogler** [comment]
@Bushuo @badnames could you please review again so we can merge this as soon as possible :D

**Review by Bushuo — APPROVED** [review]
Works nice :) Perf. is improved 👍

**Pull request merged**